src/pages/index.page.tsx

```diff
@@ -21,6 +22,18 @@ const Home: NextPage = () => {
               </div>
               <div className="text-4xl font-extrabold">Today</div>
             </section>
+            <section className="mt-36 flex flex-col items-center">
+              <Image
+                width={64}
+                height={64}
+                src="/icon/smiley.svg"
+                alt="placeholder icon"
+              />
+              <h2 className="mt-8 text-2xl font-extrabold text-center">
```

Comment:
Should we adjust the style a bit as the font size and the `line-height` is not the same as on the design

The design | Current style
--- | ---
![Screen Shot 2565-05-12 at 11 07 23](https://user-images.githubusercontent.com/1772999/167990278-3b924028-d770-4287-9d78-c14ccbc7767a.png) | ![Screen Shot 2565-05-12 at 11 08 02](https://user-images.githubusercontent.com/1772999/167990301-b4a6c7e7-41ab-44f8-9589-9fee48fc06e4.png)

Reply:
Updated text size and line height 
<img width="781" alt="Screen Shot 2565-05-12 at 16 41 08" src="https://user-images.githubusercontent.com/14077479/168041728-d4347baa-4362-4400-ab8b-30c17d2d1d9f.png">